index.html

```diff
@@ -1,14 +1,223 @@
 <!DOCTYPE HTML>
 <html>
-  <head>
-    <title>Mockup Home</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="features"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
-  </body>
+
+<head>
+  <title>Mockup Home</title>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+  <link href="styles/style.css" rel="stylesheet" type="text/css">
+  <link href="styles/general.css" rel="stylesheet" type="text/css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Racing+Sans+One&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans+Condensed:wght@300&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  
+  <section id="header">
+    <a href="index.html">
+      <div id="header-div">
+        <img id="OF" src="Images/OnlyFans.svg">
+    </a>
+    </div>
+    <br>
+    <br>
+    <hr>
+  </section>
+  <h3 id="features-title">Featured Content</h3>
+
+
+  <!-- Features section -->

```

Comment:
Good commenting and/ breaking out page sections. This is a good habit to continue to see!